src/Examples/CoreExample/Controllers/HomeController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Diagnostics;
+using CoreExample.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CoreExample.Controllers
+{
+    public class HomeController : Controller
```

Comment:
I just went with the same as the other example, should I remove Contact and About from both? 

Also went with the same layout as before with the account and home controllers - the home page isn't used, as the projects start with account/login as the initial page. I think it would make more sense to have the login page on the home page though for both Examples, and then we redirect to account once the QR code is scanned, would you agree?

Reply:
Agree on both.